package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mnet-ui-base",
   "version": "1.0.1",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
remove this change, otherwise it will fail after build on ci

Reply:
Fixed